Mesh issue with the Princeton Mouse Brain Atlas

As [reported on image.sc](https://forum.image.sc/t/princeton-mouse-brain-atlas-offset-in-3d-space/92868) there seems to be an issue with the meshes of the Princeton Mouse Brain Atlas. They have some additional transform that means they don't align with e.g. the Allen atlas.

@viktorpm @alessandrofelder is this something that was caught in the validation scripts?

Hey, yes, this does indeed fail our validation test for the root mesh and the annotation have the same bounding box:
```
AssertionError: ('Mesh coordinate 618.5 and annotation coordinate 12360.0', 'differ by more than 10 times pixel size 20.0')
```
which is not surprising, seeing as opening the mesh and the annotations in napari looks like:
![image](https://github.com/brainglobe/brainglobe-atlasapi/assets/10500965/60b85f22-0110-4bce-a96d-83801348cf5f)

--- a/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
+++ b/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
@@ -127,9 +127,9 @@ def create_atlas(working_dir, resolution):
     meshes_dir_path.mkdir(exist_ok=True)
 
     tree = get_structures_tree(structs_dict)
-    rotated_annotations = np.rot90(annotated_volume, axes=(0, 2))
+    # rotated_annotations = np.rot90(annotated_volume, axes=(0, 2))
 
-    labels = np.unique(rotated_annotations).astype(np.int32)
+    labels = np.unique(annotated_volume).astype(np.int32)
     for key, node in tree.nodes.items():
         if key in labels:
             is_label = True
@@ -153,7 +153,7 @@ def create_atlas(working_dir, resolution):
                         node,
                         tree,
                         labels,
-                        rotated_annotations,
+                        annotated_volume,
                         ROOT_ID,
                         closing_n_iters,
                         decimate_fraction,
@@ -177,7 +177,7 @@ def create_atlas(working_dir, resolution):
                     node,
                     tree,
                     labels,
-                    rotated_annotations,
+                    annotated_volume,
                     ROOT_ID,
                     closing_n_iters,
                     decimate_fraction,

--- a/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
+++ b/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
@@ -127,7 +127,6 @@ def create_atlas(working_dir, resolution):
     meshes_dir_path.mkdir(exist_ok=True)
 
     tree = get_structures_tree(structs_dict)
-    # rotated_annotations = np.rot90(annotated_volume, axes=(0, 2))
 
     labels = np.unique(annotated_volume).astype(np.int32)
     for key, node in tree.nodes.items():

--- a/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
+++ b/brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py
@@ -232,6 +232,7 @@ def create_atlas(working_dir, resolution):
         hemispheres_stack=None,
         cleanup_files=False,
         compress=True,
+        scale_meshes=True,
     )
 
     return output_filename

